validation/modelSat_collocation.py

```diff
@@ -6,18 +6,30 @@
 
 VERSION AND LAST UPDATE:
  v1.0  05/10/2022
+ v1.1  06/30/2022
 
 PURPOSE:
  Collocation/pairing ww3 field output results with altimeters.
  Matchups of ww3 results and satellite data are generated, for the same
   points (lat/lon) and time.
  Additional information is collocated as well, such as water depth,
-  distance to the nearest coast, ocean names, forecast zones, and
-  satellite names.
+  distance to the nearest coast, ocean names, forecast zones, satellite
+  names, and cyclone information.
 
 USAGE:
- This code is designed for ww3 simulations with one continuous time
-  dimension only, and not forecast with consecutive cycles or ensembles.
+ This code is designed for ww3 hindcasts or forecast with 
+  consecutive cycles (overlapped time). The ww3 file(s) are not entered
+  directly, but it is informed through a list, ww3list.txt (or any other
+  name), that is read as an argument. Multiple file names can be writen 
+  in the list, ww3 results will be appended, depending on the data 
+  structure selected (hindcast or forecast). The default is hindcast, so 
+  arrays are directly appended in time. By entering a second argument 
+  (any value greated than zero), the program assumes it is a forecast data 
+  structure, i.e., the list contains consecutive cycles (each file is 
+  one cycle). In this case, another output variable is included in the
+  final netcdf file, 'cycle', with the time of the nowcast/cycle. By
+  calculating time-cycle, you can obtain the forecast range (lead time)
+  in seconds.
  Four information must be entered by the user:
```

Comment:
Note, that just one of four is listed currently.

Reply:
I see four blocks
https://github.com/NOAA-EMC/WW3-tools/blob/9401a4094da13ecbb8365f5f3e0431a612871af7/validation/modelSat_collocation.py
I didn't enumerate them but used small dashes instead. I will replace by numbers to avoid confusion

Reply:
I see four blocks in that also.  I thought I saw something different before.  no need to replace the dashes with numbers.  I think what you have is best.